.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
The whole .idea directory should not be added to the repo.

This is IDE specific. You should use a global .gitignore to ignore that.

Reply:
Remove from code
Thanks